src/pages/home/sidebar/SubNavigation/HeaderWithBreadcrumbs.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import {View} from 'react-native';
+import PropTypes from 'prop-types';
+import _ from 'lodash';
+import Text from '../../../../components/Text';
+import styles from '../../../../styles/styles';
+import Icon from '../../../../components/Icon';
+import CONST from '../../../../CONST';
+import variables from '../../../../styles/variables';
+import PressableWithFeedback from '../../../../components/Pressable/PressableWithFeedback';
+import Header from '../../../../components/Header';
+import * as Expensicons from '../../../../components/Icon/Expensicons';
+
+const propTypes = {
+    /** Text to display for the item */
+    title: PropTypes.string,
+
+    /** Array of pressable breadcrumbs */
+    breadcrumbs: PropTypes.arrayOf(PropTypes.string),
+
+    /** Function to fire when component is pressed */
+    onPress: PropTypes.func,
+};
+
+const defaultProps = {
+    title: '',
+    breadcrumbs: [],
+    onPress: () => {},
+};
+
+const HeaderWithBreadcrumbs = React.forwardRef(({title, breadcrumbs, onPress}, ref) => (
+    <View style={[styles.flexColum]}>
+        <PressableWithFeedback
```

Comment:
Since the onPress is here, does that mean that you can't select individual parts of the breadcrums, only the whole thing is clickable?